pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -194,7 +194,7 @@ stdenv.mkDerivation ({
     runHook preSetupCompilerEnvironment
 
     echo "Build with ${ghc}."
-    export PATH="${ghc}/bin:$PATH"
+    export PATH="${if ghc.isGhcjs or false then ghcEnv else ghc}/bin:$PATH"
```

Comment:
What is the intent of this change? What is now on `PATH` that otherwise wouldn't?